exercises/verify.cpp

```diff
@@ -9,10 +9,34 @@ The number 3 is [not] present in the array.
 #include <iostream>
 using namespace std;
 
+#define EXIT_SUCCESS 0
+#define DIM 10
+
+bool checkNumber(int* arr,int num)
+{
+  for(int i=0;i<DIM;i++)
+  {
+    if(i == num )
+    {
+      return true;
+    }
+  }
+  return false;
+}
+
 int main()
 {
   // placeholder
-  int N[10] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
+  int N[DIM] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
+
+  int num;
+  cout<<"Insert number: ";
+  cin>>num;
+
+  if(checkNumber(N,num))
+    cout<<"The number "<< num <<" is present in the array \n";
+  else 
+    cout<<"The number "<< num <<" is not present in the array \n";
```

Comment:
```suggestion
ciò che cambia qui è solo il "not", potresti usare la condizione ternaria per rendere tutto più compatto:
```C++
cout << "The number " << number << " is " << checkNumber(numbers, dim, number) ? "" : "not " << "present in the array" << endl;
```
```

Reply:
purtroppo ho sbagliato a scrivere la "suggestion"